src/middleware.ts

```diff
@@ -22,7 +22,7 @@ export default async function updateSession(request: NextRequest) {
     loginUser.role === MemberRole.Member &&
     isRestrictedPath
   ) {
-    return NextResponse.redirect(new URL(Routes.Members, request.url));
+    return NextResponse.redirect(new URL(Routes.Attendance, request.url));
```

Comment:
Let's navigate user to user whatever the default page is.

Reply:
Or there's an opportunity to create a not-allowed page and redirect user there.

Reply:
Sir we will create `not-allowed` page if any route match with restricted paths.